modules/core/src/test/java/org/apache/ignite/internal/ClusterNodeMetricsSelfTest.java

```diff
@@ -75,7 +96,84 @@
         cfg.setCacheConfiguration();
         cfg.setMetricsUpdateFrequency(0);
 
-        return cfg;
+        CacheConfiguration<Integer, Object> offHeapValuesCfg = defaultCacheConfiguration();
```

Comment:
These caches are not needed for every test from the class. Create it dymically in your test's method body.